tests/pfcwd/files/pfcwd_helper.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, mg_facts, neighbors, vlan_nw):
         """
         self.mg_facts = mg_facts
         self.bgp_info = self.mg_facts['minigraph_bgp']
-        self.port_idx_info = self.mg_facts['minigraph_port_indices']
+        self.port_idx_info = self.mg_facts['minigraph_ptf_indices']
```

Comment:
does this work for non chassis system?

Reply:
Yes. there is no change for non chassis system.